network-api/networkapi/middleware.py

```diff
@@ -1,13 +1,13 @@
-from mezzanine.conf import settings
-from django.utils.deprecation import MiddlewareMixin
-
-referrer_value = 'same-origin'
-
-if settings.REFERRER_HEADER_VALUE:
-    referrer_value = settings.REFERRER_HEADER_VALUE
-
-
-class ReferrerMiddleware(MiddlewareMixin):
-    def process_response(self, request, response):
-        response['Referrer-Policy'] = referrer_value
-        return response
+from django.conf import settings
```

Comment:
Was mezzanine.conf simply importing django.conf under the hood, and then adding some stuff?